lib/EarlyFunctionAnalysis/FunctionSummaryOracle.cpp

```diff
@@ -165,18 +165,23 @@ void importModel(Module &M,
     for (const model::CallSitePrototype &CallSite :
          Function.CallSitePrototypes()) {
 
-      Oracle.registerCallSite(Function.Entry(),
-                              BasicBlockID(CallSite.CallerBlockAddress()),
-                              importPrototype(M,
-                                              ABICSVs,
-                                              CallSite.Attributes(),
-                                              CallSite.Prototype()),
-                              CallSite.IsTailCall());
+      Oracle
+        .registerCallSite(Function.Entry(),
+                          BasicBlockID(CallSite.CallerBlockAddress()),
+                          importPrototype(M,
+                                          ABICSVs,
+                                          AttributesSet(CallSite.Attributes()
+                                                          .begin(),
+                                                        CallSite.Attributes()
+                                                          .begin()),
+                                          CallSite.Prototype()),
+                          CallSite.IsTailCall());
```

Comment:
1. Split in multiple statements, same for next changes
2. You're using `.begin` twice!